DFC.Digital/DFC.Digital.Services.SendGrid/Services/MergeEmailContentService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using DFC.Digital.Data.Interfaces;
+using DFC.Digital.Data.Model;
+
+namespace DFC.Digital.Services.SendGrid
+{
+    public class MergeEmailContentService : IMergeEmailContent
+    {
+        private const string FirstNameToken = "{firstname}";
+        private const string LastNameToken = "{lastname}";
+        private const string EmailToken = "{email}";
+        private const string ContactOptionToken = "{contactoption}";
+        private const string DobToken = "{dob}";
+        private const string PostCodeToken = "{postcode}";
+        private const string ContactAdvisorQuestionTypeToken = "{contactadviserquestiontype}";
+        private const string MessageToken = "{message}";
+        private const string IsContactableToken = "{iscontactable}";
+        private const string FeedbackQuestionTypeToken = "{feedbackquestiontype}";
+        private const string TermsAndConditionsToken = "{tandc}";
+
+        public string MergeTemplateBodyWithContent(SendEmailRequest sendEmailRequest, string content)
```

Comment:
What is the difference to this and the next method? I assume we only store HTML template, hence we need to look at getting plain text from HTML field from sitefinity for this method